pkg/controller/backup_controller.go

```diff
@@ -638,6 +645,12 @@ func (c *backupController) runBackup(backup *pkgbackup.Request) error {
 				backupLog.Error(err)
 			}
 
+			err = c.checkVolumeSnapshotReadyToUse(volumeSnapshots)
+			if err != nil {
+				backupLog.Errorf("fail to wait VolumeSnapshot change to Ready: %s", err.Error())
+				return err
```

Comment:
Could you double check the impact of returning err here?
I'm not very sure, but it seems we should just log the error and let it move on?

Reply:
Modified. Record an error in backup log and move on with next backup process.